### ISSUE TITLE
Intellectual-disabled pawns crafting bloodstem items with workSpeedStat DrugProductionSpeed

### ISSUE THREAD

**Issue by ghost** [B-Knight49/Rimworld-MedicalOverhaul#14]
Pawns with intellectual work disabled (i.e. cannot craft drugs) are able to make crushed bloodstem and bloodstem bandages at the crafting spot.  Not sure if this is intended or not.

However, upon the first such pawn doing this, the debug mode returns this error:
```
Attempted to calculate value for disabled stat DrugProductionSpeed; this is meant as a consistency check, either set the stat to neverDisabled or ensure this pawn cannot accidentally use this stat (thing=McGee)
Verse.Log:Error(String)
Verse.Log:ErrorOnce(String, Int32)
RimWorld.StatWorker:GetValueUnfinalized(StatRequest, Boolean)
RimWorld.StatWorker:GetValue(StatRequest, Boolean)
RimWorld.StatWorker:GetValue(Thing, Boolean)
RimWorld.StatExtension:GetStatValue(Thing, StatDef, Boolean)
Verse.AI.<DoRecipeWork>c__AnonStorey1:<>m__1()
Verse.AI.JobDriver:DriverTick()
Verse.AI.Pawn_JobTracker:JobTrackerTick()
Verse.Pawn:Tick()
Verse.TickList:Tick()
Verse.TickManager:DoSingleTick()
Verse.TickManager:TickManagerUpdate()
Verse.Game:UpdatePlay()
Verse.Root_Play:Update()
```

It seems to be due to the defs including `<workSpeedStat>DrugProductionSpeed</workSpeedStat>`.

The pawns are able to craft the items normally after dismissing the error.

Interestingly, after the error message pops up once, I wasn't able to make it appear again (at least not in the same gameplay session) despite trying multiple times with multiple intellectual-disabled pawns.

**Comment by I-Knight-I** [comment]
Potential fix in commit 0d203ee

**Comment by ghost** [comment]
- [x] Bloodstem bandage @ crafting spot
- [x] Crushed bloodstem @ drug lab 
- [x] Crushed bloodstem @ crafting spot - pending PR #17

**Comment by I-Knight-I** [comment]
Solved in commit 6d3d765
